### DISCUSSION TITLE
JS 컴파일오류

### DISCUSSION THREAD

**Discussion by Obsihill** [naverz/zepeto-studio-kor#1112] [5. Q&A]
![image](https://user-images.githubusercontent.com/106140401/182346871-eda89524-5eec-4c52-aa6b-333e7b66bc13.png)

우측 하단 JS 컴파일이 안됩니다. 해결방법 있을까요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 타입스크립트 서버가 꺼져 있을 때는 월드 패키지를 최신 버전으로 업데이트 하시고,
유니티를 껐다가 다시 켜시면 정상적으로 동작할 수 있습니다.
ASSETS > Reimport All 또는 ZEPETO - Script - All Rebuild
을 하신 후 유니티 재시작을 해보시는 것도 방법입니다.

아래 내용을 참고하여 주세요.
https://github.com/naverz/zepeto-studio-kor/discussions/1068#discussioncomment-3226532
